content/community.html

```diff
@@ -211,6 +211,12 @@
         <td><a target="_blank" href="https://github.com/laugimethods">Laurent Magnin</a> at Logimethods</td>
         <td>Community</td>
       </tr>
+      <tr>
+        <td><a href="https://github.com/efmr/socket.io-nats" target="_blank">Socket.io-NATS Adapter</a></td>
+        <td>This library provides an adaptor to enable broadcasting of events to multiple separate socket.io server nodes through NATS.</td>
```

Comment:
noted you used "Adapter" one place, and "Adaptor" another. @chriscasper for consistency let's use "Connector" otherwise LGTM!